x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/last_value.tsx

```diff
@@ -134,7 +134,7 @@ export const lastValueOperation: OperationDefinition<LastValueIndexPatternColumn
     const hasDateFields = indexPattern && getDateFields(indexPattern).length;
     if (!hasDateFields) {
       return i18n.translate('xpack.lens.indexPattern.lastValue.disabled', {
-        defaultMessage: 'This function requires the presence of a date field in your index',
```

Comment:
I know that this is a draft, but I just randomly sneak peaked here and I wonder `index` should be replace. I think not. Maybe it's worth discussing @timroes?

Reply:
if my opinion is interesting, then it seems to me that this change is correct, lance works with `Data View` and not `indexes`. But I think it's worth clarifying this point to be sure that in other places everything is identical with that.